emission/itinerum/itinerum_boolean.py

```diff
@@ -0,0 +1,49 @@
+# Standard imports 
+import logging
+
+# Our imports
+import emission.storage.pipeline_queries as epq
+import emission.storage.timeseries.abstract_timeseries as esta
+import emission.storage.decorations.analysis_timeseries_queries as esda
+import emission.core.wrapper.itinerum_boolean as ecwib
+
+
+def set_itinerum_boolean(user_id):
+    time_query = epq.get_time_range_for_itinerum_boolean(user_id)
+    try: 
+        ibp = ItinerumBooleanPipeline()
+        ibp.user_id = user_id
+        ibp.run_itinerum_boolean_pipeline(user_id, time_query)
+        if ibp.last_trip_done is None:
+            logging.debug("After run, last_trip_done == None, must be early return")
+        epq.mark_itinerum_boolean_done(user_id, ibp.last_trip_done)
+    except:
+        logging.exception("Error while setting boolean, timestamp unchanged")
+        epq.mark_itinerum_boolean_failed(user_id)
+
+class ItinerumBooleanPipeline:
+    def __init__(self):
+        self.last_trip_done = None
+
+    @property
+    def last_trip_done(self):
+        return self._last_trip_done
+
+    def run_itinerum_boolean_pipeline(self, user_id, time_range):
+        #This is where we need to set the boolean?? 
+        #How is this done? 
+        #1.) seems to need to set the time series 
+        self.ts = esta.TimeSeries.get_time_series(user_id)
+        self.toSetFalseTrips = esda.get_entries(
+                esda.ITINERUM_BOOLEAN_KEY, user_id, time_query=time_range)
```

Comment:
Can you expand on what you mean for this? Are you referencing the four lines or the entire file itself? In this file I was attempting to set a boolean to false. For the time series stuff, I was simply replicating what I found in other implementations as I am not concrete on the time series usage quite yet.

Reply:
the first comment was about `ITINERUM_BOOLEAN_KEY`.
You can't just use `esda.ITINERUM_BOOLEAN_KEY` without actually changing `esda`.
The other timeseries code works because there are constants defined there.

The second comment is that setting a boolean to false does not achieve your goal. Your goal is to actually export the trips in a way that @kafitz can read. which means that you need to actually save the trips (and the associated locations....)

See additional comments at https://github.com/e-mission/e-mission-server/pull/823#discussion_r655677462